### PR TITLE
Add scenario for IsManagedAccount

### PR DESCRIPTION
**Pull request by JamesFerebee**

### CODE REVIEW

**Comment by kaushika-msft** [comment]
Learn Build status updates of commit _[d9acd7a](https://github.com/JamesFerebee/SupportArticles-docs/commits/d9acd7a8783a1492e112fc2c8d37f0060f3db2cc)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md) | :warning:Warning | | [Details](#user-content-c03c610694e364343a24c5e5019aea6e-e19766e7)

<a id="c03c610694e364343a24c5e5019aea6e-e19766e7"></a>
### [support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md)
 - **Line 194, Column 64**: **[Warning: hard-coded-locale - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/gmsa-troubleshooting#check-the-gmsa-account' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 40, Column 5**: **[Suggestion: disallowed-html-attribute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/disallowed-html-attribute?branch=main)]** `````HTML attribute 'style' on tag 'hr' isn't allowed. Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````
 - **Line 84, Column 5**: **[Suggestion: disallowed-html-attribute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/disallowed-html-attribute?branch=main)]** `````HTML attribute 'style' on tag 'hr' isn't allowed. Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````
 - **Line 194, Column 64**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/gmsa-troubleshooting#check-the-gmsa-account' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/07c79912-15d2-f5bd-767a-66d227fce918/202212092049163743-807/BuildReport?accessString=74a8146442c6e65ede0d5ae66dbd263a22df529cfa143720d5fc6c5dac434c42).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/07c79912-15d2-f5bd-767a-66d227fce918?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by kaushika-msft** [comment]
Learn Build status updates of commit _[a0f819d](https://github.com/JamesFerebee/SupportArticles-docs/commits/a0f819d2db4ea44420eaaaa21960fca4ac61998e)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md) | :warning:Warning | | [Details](#user-content-c03c610694e364343a24c5e5019aea6e-49941bc7)

<a id="c03c610694e364343a24c5e5019aea6e-49941bc7"></a>
### [support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md)
 - **Line 194, Column 64**: **[Warning: hard-coded-locale - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://learn.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/gmsa-troubleshooting#check-the-gmsa-account' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 194, Column 64**: **[Suggestion: docs-link-absolute - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://learn.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/gmsa-troubleshooting#check-the-gmsa-account' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/07c79912-15d2-f5bd-767a-66d227fce918/202212092053133073-807/BuildReport?accessString=b82241a12a2342b38e56dd4b6b96022e9cb1ea6bb468b130716274deba67c6e7).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/07c79912-15d2-f5bd-767a-66d227fce918?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by kaushika-msft** [comment]
Learn Build status updates of commit _[ab6df66](https://github.com/JamesFerebee/SupportArticles-docs/commits/ab6df661ecdbde70eefce8713ab9419643ec059e)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md) | :warning:Warning | | [Details](#user-content-c03c610694e364343a24c5e5019aea6e-f31f92b5)

<a id="c03c610694e364343a24c5e5019aea6e-f31f92b5"></a>
### [support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md)
 - **Line 194, Column 64**: **[Warning: file-not-found - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../virtualization/windowscontainers/manage-containers/gmsa-troubleshooting.md'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/07c79912-15d2-f5bd-767a-66d227fce918/202212092057374156-807/BuildReport?accessString=511fa06e1772e4379b9ad08c00f9ba64292286c343f49284b2da66952815a2c7).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/07c79912-15d2-f5bd-767a-66d227fce918?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by kaushika-msft** [comment]
Learn Build status updates of commit _[0e75717](https://github.com/JamesFerebee/SupportArticles-docs/commits/0e7571741f5119434e1aed7e2ec92532b738a1e8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/07c79912-15d2-f5bd-767a-66d227fce918/202212092109337739-807/BuildReport?accessString=f2d26a06f07fc5a13c72bd4630100f9015124313d3f444fd4799706bd1f90097).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/07c79912-15d2-f5bd-767a-66d227fce918?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by JamesFerebee** [comment]
I added some spacing and header lines in the doc as visually it is hard to find what section you are looking for, in my opinion. If this does not follow the template feel free to update and add more universally accepted formatting. However if we can space things out a bit, I think that would be extremely helpful for users.

**Comment by JamesFerebee** [comment]
Lab info
---------------------------------------- 
Reproed – changed registry value - HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Services\MSSQL$SQL19STAND3\ServiceAccountManaged

After restarting OS, observe error below through services.msc
 

SQL config manager: 
 
 
 
System event log error 7038  -
 
 
Log Name:      System
Source:        Service Control Manager
Date:          12/9/2022 11:03:08 AM
Event ID:      7038
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      VMREPL1.contoso.lab
Description:
The MSSQL$SQL19STAND3 service was unable to log on as CONTOSO\SQL_GMSA$ with the currently configured password due to the following error: 
The user name or password is incorrect.



Command to check registry value: sc qmanagedaccount MSSQL$SQL19STAND3
 
 
Running the command qmanagedaccount doesn’t change it as below. 
 
 
 
 
True command that changes; sc managedaccount MSSQL$SQL19STAND3 TRUE

File: support/sql/admin/error-1069-service-cannot-start.md [hunk 1]
```diff
@@ -164,14 +175,46 @@ To ensure that the service is configured properly, use the Services snap-in in M
...
 
-Type the correct password in the SQL Server service account on the SQL Server host computer. To do this, follow the procedures from [SCM Services - Change the Password of the Accounts Used](/sql/database-engine/configure-windows/scm-services-change-the-password-of-the-accounts-used).
+    Type the correct password in the SQL Server service account on the SQL Server host computer. To do this, follow the procedures from [SCM Services - Change the Password of the Accounts Used](/sql/database-engine/configure-windows/scm-services-change-the-password-of-the-accounts-used).
+
+##### Scenario 2: gMSA IsManagedAccount Flag Set Improperly
+
+    
+1. If you are using a gMSA account to run the SQL Server Service and the IsManagedAccount flag for the given service is set to false, you will receive error 7038 as soon as the cached secret is invalid.
```
Comment (PiJoCoder) [inline support/sql/admin/error-1069-service-cannot-start.md:190]:
```suggestion
1. If you are using a group Managed Service Accounts (gMSA) account to run the SQL Server Service and the IsManagedAccount flag for the given service is set to **false**, you may receive a Service Control Manager event ID 7038 as soon as the cached secret is invalid.
```

**Comment by kaushika-msft** [comment]
Learn Build status updates of commit _[138249c](https://github.com/JamesFerebee/SupportArticles-docs/commits/138249cc900842a6c205804f5b5ee7ae0aa7f429)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/07c79912-15d2-f5bd-767a-66d227fce918/202212100422070507-807/BuildReport?accessString=48191afd282a3b7bebd5ac975b901e72dc7d45836769be8a9eb753ab7539cd64).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/07c79912-15d2-f5bd-767a-66d227fce918?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Review by sevend2 — APPROVED** [review]

**Comment by kaushika-msft** [comment]
Learn Build status updates of commit _[7f49cb2](https://github.com/JamesFerebee/SupportArticles-docs/commits/7f49cb29eb2e1e821207904f6f14c28f05c66168)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[support/sql/admin/error-1069-service-cannot-start.md](https://github.com/JamesFerebee/SupportArticles-docs/blob/patch-1/support/sql/admin/error-1069-service-cannot-start.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/07c79912-15d2-f5bd-767a-66d227fce918/202212120445098399-807/BuildReport?accessString=1d420ee0a964e1bf059bd73c5bef6b9c916270b803928ef4df61549f7a3008e1).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/07c79912-15d2-f5bd-767a-66d227fce918?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Pull request merged**